### ISSUE TITLE
Redesign: Fix spacing for lesson title and sub title size

### ISSUE THREAD

**Issue by adachiu** [TheOdinProject/theodinproject#720]
This issue includes fixing the lesson page title

1. class`lesson-header__title accent` to be h3
Currently it is too big on the page: https://staging-odin-project.herokuapp.com/courses/web-development-101/lessons/gearing-up

2. Spacing between title and sub-title to be decreased to about 15px or `margin-bottom: 1.5rem;` just like the `lesson-function` card at the very bottom of the page
<img width="291" alt="screen shot 2017-09-03 at 11 45 36 pm" src="https://user-images.githubusercontent.com/13413986/30014621-1d8d09de-9102-11e7-9f8d-ed51e413efc9.png">

### Working on this task

- [ ] If you want to work on this task, please claim it by commenting below   
- [ ] Branch off the redesign branch                                                                 
- [ ] Check out our [styleguide](https://staging-odin-project.herokuapp.com/styleguide)
- [ ] When you are submitting your pull request for this task please ensure the base branch you want to merge into is the redesign branch

**Comment by chari407** [comment]
I shall work on this. Thank you.

**Comment by KevinMulhern** [comment]
Thanks @chari407 😄  I look forward to your pull request.

**Comment by chari407** [comment]
Why is it that when i start the local host and open the same page as the link provided above, I see that the design varies in both of them??

**Comment by KevinMulhern** [comment]
Hey @chari407, you'll have to switch to the redesign branch

First fetch all the remote branches
`git fetch --all`

Then switch to the redesign branch
`git checkout redesign`

**Comment by chari407** [comment]
Thank you @KevinMulhern

**Comment by KevinMulhern** [comment]
Hey @chari407 have you made any progress on this one? or do you need any help with anything?

**Comment by chari407** [comment]
Thank you @KevinMulhern. But I have actually corrected the spacing. I just have a merge conflict. I shall let you know  if I need any help. I shall get back to you once it is done.

**Comment by chari407** [comment]
Hi @KevinMulhern .. I am no long working on this. Sorry for the trouble caused. I do not want to delay it further.

**Comment by KevinMulhern** [comment]
Thanks for letting us know @chari407

**Comment by 105ron** [comment]
Maybe I could step in and do it this weekend?

**Comment by KevinMulhern** [comment]
Thanks @105ron 😄
